app/cli/status.py

```
@@ -0,0 +1,67 @@
+    json_mode: bool = typer.Option(
+        False, "--json", help="Emit machine-readable JSON instead of a table."
+    ),
+) -> None:
+    """Show progress for a flow."""
+    atelier = Atelier()
+    flow_id = _resolve_flow_id(atelier, flow_id)
+    try:
+        progress = atelier.get_status(flow_id)
+    except FileNotFoundError:
+        console.print(f"[red]unknown flow:[/red] {flow_id}")
+        raise typer.Exit(code=1)
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)